integration/base/src/main/java/com/inrupt/client/integration/base/MockAccessGrantServer.java

```diff
@@ -123,16 +140,66 @@ private void setupMocks() {
                             this.webId, this.sharedFile))));
 
         wireMockServer.stubFor(post(urlEqualTo(ISSUE))
+                    .inScenario(SCENARIO_ACCESS_GRANT)
+                    .willSetStateTo(SCENARIO_STATE_ISSUED)
                     .atPriority(1)
-                    .withHeader("Authorization", containing("Bearer"))
+                    .withHeader(HEADER_AUTHORIZATION, containing(SCHEME_BEARER))
                     .withRequestBody(containing("providedConsent"))
                     .willReturn(aResponse()
                         .withStatus(Utils.SUCCESS)
                         .withHeader(Utils.CONTENT_TYPE, Utils.APPLICATION_JSON)
                         .withBody(getResource("/vc-grant.json", wireMockServer.baseUrl(),
                             this.webId, this.sharedFile))));
 
+        wireMockServer.stubFor(post(urlEqualTo(ISSUE))
+                .atPriority(1)
+                .withHeader(HEADER_AUTHORIZATION, containing(SCHEME_BEARER))
+                .withRequestBody(containing("hasConsent"))
+                .willReturn(aResponse()
+                        .withStatus(Utils.SUCCESS)
+                        .withHeader(Utils.CONTENT_TYPE, Utils.APPLICATION_JSON)
+                        .withBody(getResource("/vc-request.json", wireMockServer.baseUrl(),
+                                this.webId, this.sharedFile))));
+
+        // Require UMA authentication for the /verify endpoint
+        wireMockServer.stubFor(post(urlEqualTo(VERIFY))
+                .atPriority(2)
+                .willReturn(aResponse()
+                        .withStatus(401)
+                        .withHeader("WWW-Authenticate", "Bearer, DPoP algs=\"ES256\", " +
+                                "UMA ticket=\"ticket-67890\", as_uri=\"" + authorisationServerUrl + "\"")));
+
+        wireMockServer.stubFor(post(urlEqualTo(VERIFY))
+                .inScenario(SCENARIO_ACCESS_GRANT)
+                .whenScenarioStateIs(SCENARIO_STATE_ISSUED)
+                .atPriority(1)
+                .withHeader(HEADER_AUTHORIZATION, containing(SCHEME_BEARER))
+                .willReturn(jsonResponse(
+                        new AccessCredentialVerification(
+                                List.of("issuanceDate", "proof", "expirationDate", "credentialStatus"),
+                                List.of(),
+                                List.of()),
```

Comment:
If we want to support this in a Java8 runtime (we don't currently but we may with Android), it would be better to use `Arrays.asList(...)` and `Collections.emptyList()`

Reply:
Addressed in https://github.com/inrupt/solid-client-java/pull/527/commits/b051b636e2e11d1d8a21411de32b103088dc59c7